jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/asserts/navigation/DesktopStepperAssert.java

```diff
@@ -62,4 +63,33 @@ public DesktopStepperAssert allStepsIncomplete() {
             Matchers.is("has all steps incomplete"));
         return this;
     }
+
+    /**
+     * Waits until step with the given number is completed.
+     *
+     * @param stepNumber step number in this stepper
+     * @return this {@link DesktopStepperAssert} instance
+     */
+    @JDIAction("Assert that step '{0}' in '{name}' is completed")
+    public DesktopStepperAssert stepCompleted(int stepNumber) {
+        SoftAssert.jdiAssert(new Timer(base().getTimeout() * 1000L)
+                        .wait(() -> element().hasStepCompleted(stepNumber - 1)) ?
+                        "has step completed" : "doesn't have step completed",
+                Matchers.is("has step completed"));
+        return this;
+    }
+
+    /**
+     * Checks if step with the given number is incomplete.
+     *
+     * @param stepNumber step number in this stepper
+     * @return this {@link DesktopStepperAssert} instance
+     */
+    @JDIAction("Assert that step '{0}' in '{name}' is incomplete")
+    public DesktopStepperAssert stepIncomplete(int stepNumber) {
+        SoftAssert.jdiAssert(element().hasStepCompleted(stepNumber - 1) ?
+                        "has step completed" : "doesn't have step completed",
```

Comment:
"has step completed" 

also duplication of the string, move it to constant, please

Reply:
Done